client/store-management-links/quick-link-category/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from '@wordpress/element';
```

Comment:
Do we need to import `React` here?

Reply:
@jeffstieler Yeah, you always have to import React when using jsx.

Soon we should be able to remove this though; https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#how-to-upgrade-to-the-new-jsx-transform